modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/socket/WebSocketManager.java

```diff
@@ -436,4 +471,124 @@ public void pingClients() {
         agents.keySet().forEach(this::ping);
         browsers.keySet().forEach(this::ping);
     }
+
+    /**
+     * @param shortName Class short name.
+     * @return Full class name.
+     */
+    protected String igniteVisor(String shortName) {
+        return VISOR_IGNITE + shortName;
+    }
+
+    /**
+     * @param taskId Task ID.
+     * @param taskCls Task class name.
+     * @param argCls Arguments classes names.
+     */
+    protected void registerVisorTask(String taskId, String taskCls, String... argCls) {
+        visorTasks.put(taskId, new VisorTaskDescriptor(taskCls, argCls));
+    }
+
+    /**
+     * Register Visor tasks.
+     */
+    @PostConstruct
+    protected void registerVisorTasks() {
+        registerVisorTask(
+            "querySql",
+            igniteVisor("query.VisorQueryTask"),
```

Comment:
Can be replaced with `VisorQueryTask.getClass().getName()`

Reply:
No, for tasks lets use strings, just for uniformity. Because several classes already not exists in current code base. And I do not like to mix java and strings.